server/src/routes/sectionsRoute.ts

```diff
@@ -0,0 +1,10 @@
+import { addSectionController, deleteSectionController, editSectionController } from '../controllers';
+import { checkAuth } from '../middlewares';
+
+const sectionsRoute = require('express').Router();
+
+sectionsRoute.post('/section', addSectionController);
+sectionsRoute.delete('/section/:id', deleteSectionController);
+sectionsRoute.put('/section/:id', editSectionController);
```

Comment:
Add checkAuth to your routes

Reply:
If you see the second line of the code, you will find it, but I didn't call it in the route to try the function in postman,
& We will add it after we start with client side

Reply:
ok